apiv2/database/models/dataset.py

```diff
@@ -81,8 +81,8 @@ class Dataset(Base):
     deposition_date: Mapped[datetime.datetime] = mapped_column(DateTime(timezone=True), nullable=False)
     release_date: Mapped[datetime.datetime] = mapped_column(DateTime(timezone=True), nullable=False)
     last_modified_date: Mapped[datetime.datetime] = mapped_column(DateTime(timezone=True), nullable=False)
-    publications: Mapped[str] = mapped_column(String, nullable=True)
+    dataset_publications: Mapped[str] = mapped_column(String, nullable=True)
```

Comment:
What do you think of having this field be `publications` across the dataset, deposition and tomogram table as the prefix is redundant? 
We could use the `CrossReferenceMixin` to keep this field and the `related_database_entries` consistent.

Reply:
Basically this PR is specifically to try to keep as much compatibility as possible with the V1 api, which was the preference that CZII had for the migration, and in the old api, the field was `dataset_publications`